tests/test_level/test_level_01.py

```diff
@@ -642,3 +642,23 @@ def test_non_keyword_with_argument_gives_invalid(self):
             extra_check_function=lambda c: c.exception.arguments['invalid_command'] == 'aks',
             max_level=17,
         )
+
+    def test_source_map(self):
+        code = textwrap.dedent("""\
+        print Hallo welkom bij Hedy!
+        forward 50
+        ask Wat is je lievelingskleur
+        echo je lievelingskleur is""")
```

Comment:
Can we add Python code in this tests? I think that is better if the Python code is next to the Hedy code and we can compare. Therefore, this will also document the Source Mapper for people who don't know how it works, which is also one of the purposes of the tests

Reply:
Great idea! [Add testing Python code to the source-mapper tests](https://github.com/hedyorg/hedy/pull/4094/commits/b8cc9d624eee1c829a71844145b96968d75ee406) fixes this.